tests/cmd/stability/main.go

```diff
@@ -219,5 +222,19 @@ func main() {
 	// truncate a sst file and check failover
 	oa.TruncateSSTFileThenCheckFailoverOrDie(cluster1, 5*time.Minute)
 
+	// stop one etcd node and k8s/operator/tidbcluster is available
+	faultEtcd := tests.SelectNode(conf.ETCDs)
```

Comment:
this `SelectNode` method will select the first etcd forever: https://github.com/pingcap/tidb-operator/blob/7b9ced3b34d57e9a9d8d1fb82e8915ced652fa87/tests/manifests/stability/stability-configmap.yaml#L19